go/database/mpt/forest_test.go

```diff
@@ -509,27 +567,36 @@ func TestForest_getAccess_Fails(t *testing.T) {
 					if err != nil {
 						t.Fatalf("failed to open forest: %v", err)
 					}
+					defer forest.Close()
 
 					// inject failing stock to trigger an error applying the update
 					var injectedErr = errors.New("failed to call Get")
 					ctrl := gomock.NewController(t)
 					cache := NewMockNodeCache(ctrl)
 					cache.EXPECT().Get(gomock.Any()).AnyTimes().Return(nil, false)
 					var n Node
+					// only the second call must fail - repeats four times for four calls
```

Comment:
I think the comment was at the right place before, cannot see why it was moved here